internal/components/kube_registry.go

```diff
@@ -0,0 +1,86 @@
+package components
```

Comment:
Similar to `KubeModel`, it would be nice to put it into a kube-specific package.

Reply:
agreed, just didn't quite get there